projects/gloo/pkg/plugins/shadowing/plugin_test.go

```diff
@@ -30,7 +54,7 @@ var _ = Describe("Plugin", func() {
 			},
 		}
 		out := &envoy_config_route_v3.Route{}
-		err := p.ProcessRoute(plugins.RouteParams{}, in, out)
```

Comment:
Would we need to test an empty RouteParams ?

Reply:
realistically the RouteParams will never be empty, we have several plugins that do a `params.Snapshot.<resourceType>.Find` without doing a nil check. This plugin now does a `Find` where it didn't used to, so thats why the test had to change to pass in a non-empty route params